1228-public_page/anonymous_user_views_landing_page_for_registration.feature

```diff
@@ -1,32 +1,32 @@
-Feature: Anonymous User View Public Page for Publication
-
-    @881
-    Scenario Outline: Anonymous User View Public Page for Publication
-        Given the Anonymous User has opened NVA
-        And they see the "<Page>"
-        When they click to view a Publication
-        Then the Public Page for Publication is opened
-        And they see page fields for
-            | Title                  |
-            | Abstract               |
-            | NPI                    |
-            | Keywords               |
-            | Publication date       |
-            | Primary language       |
-            | Projects               |
-            | Publication type       |
-            | Publication subtype    |
-            | Values per sybtype     |
-            | Clickable Contributors |
-            | Clickable emails       |
-            | Affiliations           |
-            | File download link     |
-            | ORCID                  |
-            | DOI link               |
-            | Licence                |
-            | "<Licence link>"       |
-
-        Examples:
-            | Page        | Licence link                                                            |
-            | Start Page  | https://creativecommons.org/licenses/by/4.0/deed.no                     |
-            | Public Page | https://dlr.unit.no/licenses/ntnu-alle-rettigheter-forbeholdt-forfatter |
+Feature: Anonymous User views Landing Page for Registration
+
+    @881
+    Scenario Outline: Anonymous User views Landing Page for Registration
+        Given the Anonymous User has opened NVA
+        And they see the "<Page>"
+        When they click to view a Publication
+        Then the Public Page for Publication is opened
+        And they see page fields for
+            | Title                  |
+            | Abstract               |
+            | NPI                    |
+            | Keywords               |
+            | Publication date       |
+            | Primary language       |
+            | Projects               |
+            | Publication type       |
+            | Publication subtype    |
+            | Values per sybtype     |
```

Comment:
What is this? fjern?

Reply:
tipper det var tenkt som alle felter som er knyttet til spesifikke typer?
Ikke alle typer har `peerReviewed`, `publisher`, `series` etc

Reply:
ja, det er det sikkert. 
```suggestion
            | Fields corresponding to subtype     |
```